Devscord.DiscordFramework/Framework/Commands/Services/BotCommandsPropertyConversionService.cs

```diff
@@ -12,15 +16,23 @@ public class BotCommandsPropertyConversionService
 
         public object ConvertType(string value, BotCommandPropertyType type)
         {
-            return type switch
+            try
             {
-                BotCommandPropertyType.Time => this.ToTimeSpan(value),
-                BotCommandPropertyType.Number => int.Parse(value),
-                BotCommandPropertyType.Bool => bool.Parse(value),
-                BotCommandPropertyType.UserMention => ulong.Parse(_exMention.Match(value).Value),
-                BotCommandPropertyType.ChannelMention => ulong.Parse(_exMention.Match(value).Value),
-                _ => value
-            };
+                return type switch
+                {
+                    BotCommandPropertyType.Time => this.ToTimeSpan(value),
+                    BotCommandPropertyType.Number => int.Parse(value),
+                    BotCommandPropertyType.Bool => bool.Parse(value),
+                    BotCommandPropertyType.UserMention => this.ParseToUserId(value),
+                    BotCommandPropertyType.ChannelMention => this.ParseToChannelId(value),
+                    BotCommandPropertyType.SingleWord => !value.Any(char.IsWhiteSpace) ? value : throw new InvalidArgumentsException(),
+                    _ => value
+                };
+            }
+            catch
+            {
+                throw new InvalidArgumentsException();
```

Comment:
catch and throw... 
po co?

Reply:
sorry myślałem, że tylko dla niektórych wyjątków nie wywala programu, a teraz jak spojrzałem to gdy jest wyjątek to po prostu jest komunikat